<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>showPlotsWindow</class>
 <widget class="QWidget" name="showPlotsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="sliderContainer" native="true">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>161</width>
     <height>321</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="plot">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>460</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>COMPUTE</string>
   </property>
  </widget>
  <widget class="QWidget" name="plotsContainer" native="true">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>29</y>
     <width>551</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>430</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Images</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="iidRadio">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>IID</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ccRadio">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>380</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CC</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="ccompRadio">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>400</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>CComp</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>551</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>450</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Estimated total time:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalTime">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>251</width>
     <height>18</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Estimated time remaining:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
